package.json

```diff
@@ -68,11 +68,11 @@
     "server": "npm start",
     "snyk": "snyk",
     "snyk-protect": "snyk protect",
-    "start": "npm run babel:build && http-server",
+    "start": "npm run build:components && http-server",
```

Comment:
here, the assumptions is that someone is spinning up a server, checking changes, shutting down the server... and repeat:

```bash
npm start
# ctr-c
npm start
# ctr-c
```